biggraphite/drivers/cassandra.py

```diff
@@ -1387,7 +1387,7 @@ def get_metric(self, metric_name):
             return None
         uid = result[0]
         config = result[1]
-        updated_on = result[2]
+        updated_on = datetime.datetime.fromtimestamp(result[2]/1000)
```

Comment:
Cannot we reproduce the issue with a test and only then fix it?

Reply:
Done